src/BinaryParsers/ProgramDatabase/Pdb.cs

```diff
@@ -275,7 +275,18 @@ public bool IsSegmentWithIdWritable(uint addressSection)
         private HashSet<uint> GenerateExecutableSectionContribIds()
         {
             var result = new HashSet<uint>();
-            var enumSectionContribs = CreateDiaTable<IDiaEnumSectionContribs>();
+            IDiaEnumSectionContribs enumSectionContribs = null;
+
+            try
+            {
+                CreateDiaTable<IDiaEnumSectionContribs>();
```

Comment:
ouch, no, no warning. all this indicates we have missing code coverage in tests, will track that down. updated PR. to be consistent I applied this pattern to another caller to CreateDiaTable<t>